src/snyk/snykCode/views/suggestion/codeSuggestionWebviewProvider.ts

```diff
@@ -227,47 +242,51 @@ export class CodeSuggestionWebviewProvider extends WebviewProvider implements IC
           </div>
           <div id="example"></div>
         </section>
-        <section class="feedback-section delimiter-top">
-          <div id="ignore-section">
-            <div id="ignore-top">Do you want to hide this suggestion from the results?</div>
-            <div class="ignore-actions row">
+        <section class="delimiter-top">
+          <div id="actions-section">
+            <div class="actions row">
+              ${
+                this.configuration.getPreviewFeatures().reportFalsePositives
+                  ? `
```

Comment:
Was not easy to read, but okay - maybe a bit more verbosity, if possible.

Reply:
It's how autoformatting made it, don't like it either. But since this shouldn't live for too long as it's a feature flag switch don't bother much.